pallets/capacity/src/benchmarking.rs

```diff
@@ -59,7 +59,7 @@ benchmarks! {
 		let block_number = 4u32;
 
 		let mut staking_account = StakingAccountDetails::<T>::default();
-		staking_account.increase_by(10u32.into());
+		staking_account.increase_by(500u32.into());
```

Comment:
Increase token amount to resolve an underflow error in the withdraw_unstaked benchmarking